### PR TITLE
NN-2936 filter adjudications by finding code

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java [hunk 1]
```diff
@@ -94,6 +94,7 @@
     ResponseEntity<AdjudicationSearchResponse> getAdjudicationsByOffenderNo(@ApiParam(value = "offenderNo", required = true, example = "A1234AA") @PathVariable("offenderNo") @NotNull String offenderNo,
                                           @ApiParam(value = "An offence id to allow optionally filtering by type of offence") @RequestParam(value = "offenceId", required = false) String offenceId,
                                           @ApiParam(value = "An agency id to allow optionally filtering by the agency in which the offence occurred") @RequestParam(value = "agencyId", required = false) String agencyId,
+                                          @ApiParam(value = "Finding code to allow optionally filtering by type of finding") @RequestParam(value = "finding", required = false) String finding,
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java:97]: would be good to add an example etc. here, I've no idea what a `finding` is

**Review by Nimphal — DISMISSED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**